include/core/parameters.h

```diff
@@ -1864,6 +1864,13 @@ namespace Parameters
       {
         return vector_of_string;
       }
+    else
+      {
+        AssertThrow(false,
+                    ExcMessage("Error: convert_string_to_vector only works for "
+                               "int, double and string types."));
+      }
```

Comment:
```suggestion

        AssertThrow(false,
                    ExcMessage("Error: convert_string_to_vector only works for "
                               "int, double and string types."));
      
```

If you have reached this point there is nothing to be done, so there's no need for an if statement.